utils/verify_intel_gaudi/verify_intel_gaudi_installation.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+# This playbook need to be called during the default omnia provisioning
+# of Gaudi devices. This will make sure the device will function properly
+# and help early detection of HW/SW missing requirements.
+# It's recommended to do extensive tests when installing a new Gaudi node.
+# This can be controlled using var: verify_intel_gaudi_with_sanity_test_only: false
+
+- name: Verify Intel Gaudi installation
+  hosts: slurm_node, kube_node
```

Comment:
we dont support slurm on ubuntu, can we remove slurm_node host from here?

Reply:
done